package.json

```
@@ -46,11 +46,17 @@
     "lodash": "^4.17.15"
   },
   "resolutions": {
+    "eslint": "^8.25.0",
     "kind-of": "6.0.3",
     "glob-parent": "^5.1.2",
+    "marked": "^4.0.14",
     "node-notifier": "^9.0.0",
+    "react": "^18.2.0",
```

We are very specific with our react version, I wouldn't want to unintentionally cause a bump by setting a resolution with a range.

```suggestion
    "react": "18.2.0",
```

---

The change in this file is reverted.